website/docs/developers/circuits.md

```diff
@@ -233,11 +233,33 @@ proof types:
 - Protocol compliance validation
 - Regression testing for circuit changes
 
-## Circuit Constraint Extraction
+### Circuit Generation Process
+
+Since these constraint capabilities are missing, mina rust nodes requires
+externally generated circuit data. The following process describes how circuits
+are created and distributed using the original Mina codebase:
 
-For a comprehensive technical overview of circuit constraint extraction, see the
-[circuit_blobs module documentation](https://o1-labs.github.io/mina-rust/api-docs/ledger/proofs/circuit_blobs/index.html)
-in the ledger crate.
+<!-- TODO:
```

Comment:
As we do on the release process page, you can add a block "alert warning" to have something like this: 

<img width="1018" height="366" alt="Image" src="https://github.com/user-attachments/assets/33e362a7-8f72-4d43-89a5-f65d00067816" />.

Could you update please? It informs the reader that it is still in progress. Comments are not visible to readers.